airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -319,6 +330,19 @@ def post_dag_run(*, dag_id: str, session: Session = NEW_SESSION) -> APIResponse:
                 dag_hash=get_airflow_app().dag_bag.dags_hash.get(dag_id),
                 session=session,
             )
+            try:
+                to_log_data = {
+                    "event": "rest_dag_trigger",
+                    "owner": g.user.username,
+                    "extra": None,
+                    "task_id": None,
+                    "dag_id": f"{dag_run.dag_id}",
```

Comment:
With the `action_logging`, do we still need this?

Reply:
The point was to write the `log` table. To do so we first need to create _Log_ object. That's what we actually do here.
We don't write to `dag_run` table, thus cannot use _DagRun_ object as itself.
And also, owner is only accessible from `flask.g`.
So we need to "construct" _Log_ object

Reply:
Can you look at the code for the `action_logging`? https://github.com/apache/airflow/blob/be0e35321f0bbd7d21c75096cad45dbe20c2359a/airflow/www/decorators.py#L105-L112